boards/others/elemrv/doc/index.rst

```diff
@@ -0,0 +1,119 @@
+.. _elemrv-n:
+
+Overview
+********
+
+ElemRV-N is an end-to-end open-source RISC-V microcontroller designed using SpinalHDL.
+
+Version 0.2 of ElemRV-N was successfully fabricated using `IHP's Open PDK`_, a 130nm open semiconductor process, with support from `FMD-QNC`_.
+
+For more details, refer to the official `GitHub Project`_.
+
+.. note::
+   The currently supported silicon version is ElemRV-N 0.2.
+
+
+Supported Features
+******************
+
+.. todo:: add board-supported-hw with first interface
```

Comment:
Why not add it now?

Reply:
Documentation failed to build because of `board-supported-hw directive must be used in a board documentation page.`. I re-added it to take look again.

Maybe @kartben has an idea why it fails?

Reply:
You're missing something like this https://github.com/zephyrproject-rtos/zephyr/blob/76e1fc7713a4a3f2b50c497afddec0364a34c10b/boards/others/black_f407ve/doc/index.rst?plain=1#L1

I would recommend using the board documentation template https://github.com/zephyrproject-rtos/zephyr/blob/76e1fc7713a4a3f2b50c497afddec0364a34c10b/doc/templates/board.tmpl

Reply:
Thanks!